examples/bri-3/src/bri/identity/bpiSubjects/capabilities/getBpiSubjectById/getBpiSubjectByIdQuery.handler.ts

```diff
@@ -0,0 +1,21 @@
+import { IQueryHandler, QueryHandler } from "@nestjs/cqrs";
+import { BpiSubjectDto } from "../../api/dtos/response/bpiSubject.dto";
+import { BpiSubjectRepository } from "../../persistence/bpiSubjects.repository";
+import { GetBpiSubjectByIdQuery } from "./getBpiSubjectById.query";
+
+@QueryHandler(GetBpiSubjectByIdQuery)
+export class GetBpiSubjectByIdQueryHandler implements IQueryHandler<GetBpiSubjectByIdQuery> {
+  constructor(
+    private readonly repo: BpiSubjectRepository,
+  ) {}
+
+  async execute(query: GetBpiSubjectByIdQuery) {
+    const bpiSubject = await this.repo.getBpiSubjectById(query.id);
+    return { // TODO:  // TODO: Write generic mapper domainObject -> DTO
```

Comment:
lets make issue for generic mapper?

Reply:
Whoever has time first.

Reply:
This task is very important and therefore requires a double TODO.